### PR TITLE
Fix #707.

### LINKED ISSUE #707: Readme needs to be updated
Not sure if it should be posted in issues or pull requests, but since I did not understood how to post in pull requests (I'm new to github), I'm doing it here.
In my first attempts to build the client by myself for the first time I encountered a few issues.
I managed to do it in the end with some help and one of the issues is solved already but there are things in the Readme that need to be updated:
Tere are 2 issues there:
1.- Under the "To configure manually with CMake and build:" section, line 6 say: "Set the CMAKE_INSTALL_PREFIX option under CMAKE to the cwe-prefix folder that you extracted from the development libraries bundle." There is no cwe-prefix folder in the devlibs anymore, so this needs to be changed.
2.- Under the "Running Instructions" section, line 2 say: "Copy the Python and SDL folders from Scripts into the MOUL-OS folder, as well as the files in dat into the existing dat folder." The location is wrong, since the 3 folders are in the bin folder.
So I think the Readme need to be updated.

### PR DESCRIPTION
**Pull request by Hoikas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,9 +78,9 @@ Currently, compilation only targets Windows systems and requires Visual Studio
    aforementioned location called *build*.
 4. Check the **Grouped** and **Advanced** options.
 5. Press **Configure**. Select *Visual Studio 15 2017* as the generator.
-6. Set the *CMAKE_INSTALL_PREFIX* option under CMAKE to the *cwe-prefix* folder
-   that you extracted from the [development libraries
-   bundle](https://github.com/H-uru/PlasmaPrefix/releases/download/2020.05.01/devlibs.zip).
+6. Set the *CMAKE_INSTALL_PREFIX* option under CMAKE to the location that you
```
Comment (dpogue) [inline README.md:81]: The phrasing "location that you extracted" sounds fine spoken, but "location where you extracted" might be clearer in written form.

Comment (Hoikas) [reply README.md]: Fixed

File: README.md [hunk 2]
```diff
@@ -97,7 +97,7 @@ fully-patched installation of MOULa provided by Cyan Worlds.
 1. Copy the **files from your existing MOULa installation** to a new folder, or
    install fresh if you do not already have it installed. This folder will be
    referred to as *MOUL-OS* for the remainder of these instructions.
-2. Copy the **Python and SDL** folders from Scripts into the *MOUL-OS*
+2. Copy the **Python and SDL** folders from the `<build_dir>\bin` into the *MOUL-OS*
```
Comment (dpogue) [inline README.md:100]: Either remove "the" before the dir name or add "folder" after it.

File: README.md [hunk 3]
```diff
@@ -78,9 +78,9 @@ Currently, compilation only targets Windows systems and requires Visual Studio
    aforementioned location called *build*.
 4. Check the **Grouped** and **Advanced** options.
 5. Press **Configure**. Select *Visual Studio 15 2017* as the generator.
-6. Set the *CMAKE_INSTALL_PREFIX* option under CMAKE to the *cwe-prefix* folder
-   that you extracted from the [development libraries
-   bundle](https://github.com/H-uru/PlasmaPrefix/releases/download/2020.05.01/devlibs.zip).
+6. Set the *CMAKE_INSTALL_PREFIX* option under CMAKE to the location where you
+   extracted the [development libraries bundle]
+   (https://github.com/H-uru/PlasmaPrefix/releases/download/2020.05.01/devlibs.zip).
```
Comment (zrax) [inline README.md:83]: the `]` needs to be right next to `(` (i.e. no whitespace or newline) in order for this to render correctly as a link

**Pull request merged**